samples/dfu/ab/src/ab_utils.c

```diff
@@ -162,14 +162,14 @@ static void toggle_slot_for_single_boot(void)
 {
 	int err = 0;
 	enum ab_boot_slot active_slot = active_boot_slot_get();
-	enum ab_boot_slot new_slot = SLOT_INVALID;
+	enum boot_slot new_slot = BOOT_SLOT_NONE;
 
 	if (active_slot == SLOT_A) {
 		LOG_INF("Temporarily switching slots (A -> B)");
-		new_slot = SLOT_B;
+		new_slot = BOOT_SLOT_SECONDARY;
```

Comment:
I won't block it but can't we do a `#define SLOT_A BOOT_SLOT_PRIMARY`? Note that `SLOT_A` is used in the line above